fastlane/Fastfile

```diff
@@ -55,21 +55,23 @@ platform :ios do
     crashlytics(
       crashlytics_path: './Pods/Crashlytics/',
       groups: options[:group],
-      notifications: 'true'
+      notifications: 'true',
+      notes: options[:notes]
     )
   end
-  
+
   desc "Build and upload to Crashlytics with (configuration (Release)), (include_bitcode (false)), (export_method (enterprise)), and (group)."
   lane :beta do |options|
     options[:configuration] ||= "Release"
     options[:include_bitcode] ||= false
     options[:export_method] ||= "enterprise"
-    
+    options[:notes] ||= changelog_from_git_commits
+
     raise "beta: A Crashlytics group must be passed as an option." unless options[:group]
-    
+
     build(configuration: options[:configuration], include_bitcode: options[:include_bitcode], export_method: options[:export_method])
-    
-    uploadToCrashlytics(group: options[:group])
+
+    uploadToCrashlytics(group: options[:group], notes: options[:notes])
```

Comment:
Again, if the user doesn't pass any notes, we don't want to call this with a notes parameter.